changelog.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v5.0.5...HEAD)
 # [v5.0.5](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v5.0.4...v5.0.5)
 ### Fixed
+* [PCM-1505](https://inindca.atlassian.net/browse/PCM-1505) fix changing camera while muted breaks device selector.
```

Comment:
Could you move this under the **Unreleased** header? `v5.0.5` has already been released so your changes would fall under **Unreleased** until we get a new version out.